src/zos_files/zowe/zos_files_for_zowe_sdk/datasets.py

```diff
@@ -213,96 +233,99 @@ def blksize(self, blksize: Optional[int]):
 
     @property
     def lrecl(self) -> Optional[int]:
+        """Get the length of logical records."""
         return self.__lrecl
 
     @lrecl.setter
     def lrecl(self, lrecl: Optional[int]):
+        """Set the length of logical records."""
         self.__lrecl = lrecl
 
     @property
     def storclass(self) -> Optional[str]:
+        """Get the storage class."""
         return self.__storclass
 
     @storclass.setter
     def storclass(self, storclass: Optional[str]):
+        """Set the storage class."""
         self.__storclass = storclass
 
     @property
-    def mgntclass(self) -> Optional[str]:
-        return self.__mgntclass
+    def mgmtclass(self) -> Optional[str]:
+        """Get the management class."""
+        return self.__mgmtclass
 
-    @mgntclass.setter
-    def mgntclass(self, mgntclass: Optional[str]):
-        self.__mgntclass = mgntclass
+    @mgmtclass.setter
+    def mgmtclass(self, mgmtclass: Optional[str]):
+        """Set the management class."""
+        self.__mgmtclass = mgmtclass
 
     @property
     def dataclass(self) -> Optional[str]:
+        """Get the data class."""
         return self.__dataclass
 
     @dataclass.setter
     def dataclass(self, dataclass: Optional[str]):
+        """Set the data class."""
         self.__dataclass = dataclass
 
     @property
     def dsntype(self) -> Optional[str]:
+        """Get the type of dataset."""
         return self.__dsntype
 
     @dsntype.setter
     def dsntype(self, dsntype: Optional[str]):
+        """Set the type of dataset."""
         self.__dsntype = dsntype
 
     @property
     def like(self) -> Optional[str]:
+        """Get the dataset name to copy attributes from."""
         return self.__like
 
-    def to_dict(self) -> dict:
-        """Return the DatasetOption as a dict
+    @like.setter
+    def like(self, like: Optional[str]):
+        """Set the dataset name to copy attributes from."""
+        self.__like = like
```

Comment:
Curious why we're adding a setter for `like` in this PR? I believe that beforehand we decided against having a setter for it, because it would require reinitializing the class to update default values of other properties.